database/migrations/sqls/20240715163245-initial-db-up.sql

```diff
@@ -0,0 +1,77 @@
+DROP TABLE IF EXISTS users CASCADE;
+DROP TABLE IF EXISTS admins;
+DROP TABLE IF EXISTS bidder CASCADE;
+DROP TABLE IF EXISTS skill CASCADE;
+DROP TABLE IF EXISTS contractor CASCADE;
+DROP TABLE IF EXISTS bids CASCADE;
+DROP TABLE IF EXISTS tender CASCADE;
+DROP TABLE IF EXISTS tender_skill CASCADE;
+DROP TABLE IF EXISTS bidder_skill CASCADE;
+
+
+CREATE TABLE users (
+    id SERIAL PRIMARY KEY,
+    email VARCHAR(120) NOT NULL UNIQUE,
+    password_hash VARCHAR(60) NOT NULL,
+    user_type VARCHAR(30) NOT NULL
+);
+
+CREATE TABLE admins (
+    id SERIAL PRIMARY KEY,
+    first_name VARCHAR(100),
+    last_name VARCHAR(100),
+    cyf_role VARCHAR(100),
+    last_update TIMESTAMP,
+    user_id INTEGER REFERENCES users(id)
+);
+
+CREATE TABLE bidder (
+    user_id INTEGER PRIMARY KEY,
+    user_name VARCHAR(100),
+    first_name VARCHAR(100),
+    last_name VARCHAR(100),
+    last_update TIMESTAMP,
+    FOREIGN KEY (user_id) REFERENCES users(id)
+);
+
+CREATE TABLE skill (
+    skill_id SERIAL PRIMARY KEY,
+    skill_name VARCHAR(50)
+);
+
+CREATE TABLE contractor (
```

Comment:
Let's use the term `buyer`, since that's the term used in the requirements document

Reply:
Alright. I will fix it.